snp_application/config/snp.btproj

```diff
@@ -24,6 +24,16 @@
             <input_port name="action_name" default="follow_joint_trajectory"/>
             <input_port name="trajectory" default="{trajectory}"/>
         </Action>
+        <Action ID="FreespaceMotionPlanPub" editable="true">
```

Comment:
Not sure why we need another trajectory publisher specifically for freespace. If this is specific for the process + freespace return to start motion then maybe we just make another internal BT action that concatenates two trajectories and then we visualize that one.

Reply:
> maybe we just make another internal BT action that concatenates two trajectorie

This already exists as CombineTrajectories